docs/examples/ml_training.rst

```diff
@@ -2,4 +2,9 @@
 ML Model Training
 ====================
 
+Hyperparameter Optimization is a key part of training machine learning models. For more sophisticated examples that
+aren't grid search, `burr` is a quick and easy way to implement hyperparameter optimization. You want to keep
```

Comment:
It's probably just as good for grid search TBH. Also, no parallelism. This is a good reason to make parallelism/multiple async steps a first-class citizen later on.